app/konnect/configure/runtime-manager/runtime-groups/index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Runtime Groups
+no_version: true
+---
+
+Konnect manages runtime configuration in runtime groups. Each runtime group acts
+as a separate control plane and can manage runtime configurations independently
+of any other group. You can find a list of all runtime groups in your organization
+on the [Runtime Manager overview](https://cloud.konghq.com/runtime-manager/).
+
+* A **runtime group** is a collection of API connectivity runtime instances
+sharing the same configuration and behavior space.
+
+* A **runtime instance** is a single data plane node with a single instance of
+a runtime, such as Kong Gateway. Runtime instances service traffic for the runtime
+group. All runtime instances in one runtime group
+must be of the same type. Currently, only Kong Gateway runtime types are supported.
+
+Every organization has one default runtime group. With an Enterprise subscription,
+you can configure two additional custom runtime groups in the same Konnect
+account to manage runtime instances and their configuration in any groupings
+you want.
+
+## Default runtime group
+
+The default runtime group is the foundational group in Konnect. Every
+organization starts with one default group.
+
+This group can't be renamed or deleted, and its status as the default
+group can't be changed.
+
+### Application registration in the Dev Portal
+
+When publishing documentation from the ServiceHub to the Dev Portal, you can
+publish API specs from any version -- no matter which runtime group the version
+is in. However, application registration is only supported for Service versions
+running in the default runtime group.
```

Comment:
Do we have any practices in our documentation where we point out a known limitation, but also note that it is in our roadmap to be addressed?

This limitation is a notably large one, so while this is accurate for the helium release, it would be nice if there's a way for prospects to contact-us for updates etc.

Reply:
No, we don't have anything like that. We do point out limitations, but otherwise if a feature isn't in a steady state, we label is as "beta". We try not to make any promises or set expectations for roadmaps in the docs, or at least, we've avoided that in the past.

@mheap do you have a preference for handling this kind of issue?

Reply:
Unless we have a committed date, I tend not to add upcoming features to the docs. Adding "we intend to add support for all runtime groups, but do not have an ETA" doesn't help anyone in this instance imo